releng/release-notes-next/sysusers-handled-by-rpm.feature

```diff
@@ -0,0 +1,4 @@
+New Mock RPM package provides the systemd-sysusers drop-in configuration files automatic
+`mock 135` group ID allocation.
+https://docs.fedoraproject.org/en-US/packaging-guidelines/UsersAndGroups/#_dynamic_allocation
+
```

Comment:
nits: Trailing newline, "s/automatic/for automatic/", "s/files/file/"

We are also using this: https://rpm-software-management.github.io/rpm/manual/users_and_groups.html